project-charter.md

```diff
@@ -20,25 +20,25 @@ The mission of the Project is to:
+5.  Workgroups are estabished and maintained by Project contributors to advance Project work. Workgroups focus on specific metric, methodological, ethical, and technical issues associated with open source project health. 
 
-6.  Roles: Technical Committee projects will generally involve Contributors and Maintainers:
+6.  Roles: Committees and Workgroups involve Contributors and Maintainers:
 
-    1.  Contributors: anyone in the community that contributes code, documentation, use cases, standardized metrics, or other technical artifacts to the Project;
+    1.  Contributors: anyone in the community that contributes code, documentation, use cases, standardized metrics, or other community activities related to the Project;
 
-    2. Maintainers: Contributors who have the ability to commit directly to a Project’s main branch, repository, or wiki page and are responsive to contributions or changes from Contributors.
+    2. Maintainers: Contributors who have the ability to commit directly to a Project’s main repository branch and are responsive to contributions or changes from Contributors. Maintainers are listed in the README of each repository. 
 
-7.  Technical participation in the Project, including technical contributions to any Technical Committee, is open to all under the terms of this Charter. The initial Maintainers will be the list of Maintainers identified in the Project’s code repository or wiki site.
+7.  Participation in the Project, including contributions to any Committee or Workgroup, is open to all under the terms of this Charter. 
 
-8.  The GB may (1) establish work flow procedures for the submission, approval, and closure/archiving of Technical Committees and projects of Technical Committees, (2) set requirements for the promotion to or removal from Technical Committee roles, as applicable, and (3) amend, adjust, refine and/or eliminate the roles as it sees fit.
+8.  The GB may (1) establish work flow procedures for the submission, approval, and closure/archiving of Committees and projects of Committees, (2) set requirements for the promotion to or removal from Committee roles, as applicable, and (3) amend, adjust, refine and/or eliminate the roles as it sees fit.
```

Comment:
Maybe the same for workgroups?

Reply:
Depends on what we decide regarding comment on line 31.

Reply:
If we agree on wgs being only disbanded by the GB, this wouldn't be needed.

Reply:
This comment was about the responsibility the GB has over Working Groups. Jesus suggested to assign the same responsibilities as for Committees.